apps/emqx_durable_storage/test/emqx_ds_storage_bitfield_lts_SUITE.erl

```diff
@@ -177,20 +177,33 @@ t_new_generation_inherit_trie(_Config) ->
     ?check_trace(
         begin
             %% Create a bunch of topics to be learned in the first generation
-            Timestamps = lists:seq(1, 10_000, 100),
-            Batch = [
-                begin
-                    Topic = emqx_topic:join(["wildcard", integer_to_binary(I), "suffix", Suffix]),
-                    {TS, make_message(TS, Topic, integer_to_binary(TS))}
-                end
+            Batch1 = [
+                {TS, make_message(TS, make_topic(wildcard, I, suffix, Suffix), bin(TS))}
              || I <- lists:seq(1, 200),
-                TS <- Timestamps,
+                TS <- lists:seq(0, 1_000, 10),
```

Comment:
It's pretty confusing that `TS` is used for both timestamp and as a topic level. I think it's better to use a constant timestamp for all messages (I believe it's irrelevant in this test), and rename `TS` to `J`.

Reply:
It's actually not used as a topic level, but it indeed should be irrelevant.